website/sidebars.js

```diff
@@ -468,10 +469,11 @@ const sidebarSettings = {
         "docs/collaborate/collaborate-with-others",
         {
           type: "category",
-          label: "Explore dbt projects",
+          label: "Navigate dbt Explorer",
```

Comment:
confirm with cameron and azzam if that reads ok?

Reply:
@cafzal how do you feel about adding explorer to the sidebar of the docs: 'navgiate dbt explorer'